internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -227,7 +228,7 @@ func (r *Reconciler) reconcileControlPlane(ctx context.Context, s *scope.Scope)
 			},
 		},
 	}); err != nil {
-		return errors.Wrapf(err, "failed to update %s", tlog.KObj{Obj: s.Desired.ControlPlane.Object})
+		return errors.Wrapf(err, "failed to reconcile %s", tlog.KObj{Obj: s.Desired.ControlPlane.Object})
```

Comment:
Do we also only have to log the kind here?

Reply:
Will tlog not take care of that with it's String() method?

Reply:
tlog.Kobj logs the kind and the name, but the name is random which gives as the reconcile with random errors problem here.

So I was suggesting to use `return errors.Wrapf(err, "failed to reconcile %s", s.Desired.ControlPlane.Object.GetKind())` instead (like we do now in similar cases below)

Reply:
We could also just drop the whole error here, given that we produce more or less the same but more precise inside of `reconcileReferencedObject`

Reply:
I get you now - I'll just return the errors as they are from here.